### ISSUE TITLE
[Teamserver-Client--Bug]: 

### ISSUE THREAD

**Issue by Bigcityjoe17** [HavocFramework/Havoc#513]
### What happened?

Updates to golang dependencies are causing installation failure. I've tried many things but it wont work.
When adding additional dependencies: 
go mod download golang.org/x/sys
go mod download github.com/ugorji/go

This is the returned error:
/root/Havoc/teamserver/go.mod:3: invalid go version '1.21.0': must match format 1.23
/root/Havoc/teamserver/go.mod:5: unknown directive: toolchain

### Did You Do a Pull First?

Latest (You performed a pull first)

### Did You Try With the Dev Branch?

No (You only tried the main branch...)

### Relevant log output

```shell
/root/Havoc/teamserver/go.mod:3: invalid go version '1.21.0': must match format 1.23
/root/Havoc/teamserver/go.mod:5: unknown directive: toolchain
```

### Did You Read Over Your Issue First?

- [X] I declare I made an effort and provided the necessary information for replication of the issue.

**Comment by Bigcityjoe17** [comment]
the dev branch works. Its only the main branch with the latest update thats affected.

**Comment by Cracked5pider** [comment]
check again. I reverted the go version back to go 1.18 (fixed in: 69ce17cb3686641e92a657cece5989feb9af64ed)

**Comment by goheesheng** [comment]
Having the same issue here
